src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -150,13 +160,57 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
         <template is="dom-repeat" items="[[_entities]]" as="entity">
           <tr>
             <td>
-              <paper-icon-button
-                on-click="entityMoreInfo"
-                icon="hass:information-outline"
-                alt="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
-                title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
-              >
-              </paper-icon-button>
+              <paper-menu-button close-on-activate="true">
+                <paper-icon-button
+                  icon="hass:dots-vertical"
+                  slot="dropdown-trigger"
+                  alt="menu"
+                ></paper-icon-button>
+                <paper-listbox
+                  slot="dropdown-content"
+                  role="listbox"
+                  selected="{{selectedItem}}"
+                >
+                  <paper-item on-click="entityMoreInfo">
```

Comment:
Use `paper-icon-item` instead of `paper-icon-button`:
```html
<paper-icon-item>
    <ha-icon icon="hass:information-outline" slot="item-icon"></ha-icon>
    [[localize('ui.panel.developer-tools.tabs.states.more_info')]]
</paper-icon-item>
```

Reply:
Items replaced :)